UI/window-basic-preview.cpp

```diff
@@ -416,6 +415,19 @@ void OBSBasicPreview::keyReleaseEvent(QKeyEvent *event)
 
 void OBSBasicPreview::mousePressEvent(QMouseEvent *event)
 {
+	if (scrollMode && GetScalingMode() != ScalingMode::Window &&
+		event->button() == Qt::LeftButton) {
```

Comment:
Style nitpick:  I _think_ the indent for multi-line if statement conditions is four spaces (to match "if (").  See lines 383-384 as an example of this.

Side note:  I do not know how that works with "else if" statements.